packages/world/src/WorldResourceId.sol

```diff
@@ -92,7 +92,7 @@ library WorldResourceIdInstance {
       string(
         abi.encodePacked(
           resourceNamespace == ROOT_NAMESPACE ? ROOT_NAMESPACE_STRING : resourceNamespace,
-          "_",
+          "/",
           resourceName == ROOT_NAME ? ROOT_NAME_STRING : resourceName,
           ".",
           resourceType
```

Comment:
I think this is leftover from our "file" analogy, which I think we've moved on from?

We should agree on a stringified version of these resource IDs, and my vote would be something like
```
type_namespace_name
type:namespace:name
```
(Stripe uses e.g. `acct_1234` and `ch_1234`)

Reply:
don't have a strong opinion on the separator string, `_` was just a bit confusing in `ROOT_NAMESPACE_ROOT_NAME`. `:` works for me!

Reply:
how about `namespace:name:type`? (to make the order match the encoding)

Reply:
I would still argue for the more human readable form here, and asked in the other PR about moving the order for our helpers

Reply:
from discord for future reference

> I’m fine sticking with encoding order for resources
> not my favorite but I’m optimizing for consistency and gas